web/src/settings_data.ts

```diff
@@ -253,3 +253,13 @@ export function user_email_not_configured(): boolean {
     // page_params.demo_organization_scheduled_deletion_date
     return page_params.is_owner && page_params.delivery_email === "";
 }
+
+export function type_id_to_string(type_id: number): string | undefined {
+    const bot_type = page_params.bot_types.find((bot_type) => bot_type.type_id === type_id);
+
+    if (bot_type === undefined) {
+        return undefined;
+    }
+
+    return bot_type.name;
+}
```

Comment:
I am not sure whether this is the best place to have this function. Other options are `settings_config` or `bot_data`, `settings_config` is mostly used for storing value of constants, so might be a good place for this and `bot_data` is used to get data related to a bot which is not the case here.

Reply:
I think this location is fine but we should call it `bot_type_id_to_string`